scenario/scenario_shacl.ttl

```diff
@@ -0,0 +1,247 @@
+            sh:order 0 ;
+            sh:path scenario:numberTrafficObjects ],
+        [ skos:example "myDriver, BestPedestrianController2000" ;
+            sh:datatype xsd:string ;
+            sh:description "Usage of simulation Core internal or external controllers."@en ;
+            sh:message "Validation of controllers failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "controllers"@en ;
+            sh:order 5 ;
+            sh:path scenario:controllers ],
+        [ skos:example "8" ;
+            sh:datatype xsd:unsignedInt ;
+            sh:description "Count of traffic permanent objects"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of permanentTrafficObjects failed!"@en ;
+            sh:name "permanentTrafficObjects"@en ;
+            sh:order 2 ;
+            sh:path scenario:permanentTrafficObjects ],
+        [ skos:example "via trajectory, via support points" ;
+            sh:datatype xsd:string ;
+            sh:description "Type of movement used by traffic participants according to the OpenSCENARIO standard."@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of movementDescription failed!"@en ;
+            sh:name "movementDescription"@en ;
+            sh:order 4 ;
+            sh:path scenario:movementDescription ],
+        [ skos:example "stopIfTTCLowerThan" ;
+            sh:datatype xsd:string ;
+            sh:description "Scenario/domain specific commands or actions"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of customCommands failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "customCommands"@en ;
+            sh:order 6 ;
+            sh:path scenario:customCommands ] ;
+    sh:targetClass scenario:Participants .
+
+scenario:StructuralShape a sh:NodeShape ;
+    sh:property [ skos:example "http://mywebsite.de/Catalogs" ;
+            sh:description "Link to Catalogs"@en ;
+            sh:message "Validation of catalogs failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "catalogs"@en ;
+            sh:node general:LinkShape ;
+            sh:order 0 ;
+            sh:path scenario:catalogs ],
+        [ skos:example "http://mywebsite.de/myModel.fbx" ;
+            sh:description "Link to 3D models"@en ;
+            sh:message "Validation of 3dModels failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "3dModels"@en ;
+            sh:node general:LinkShape ;
+            sh:order 1 ;
+            sh:path scenario:3dModels ],
+        [ skos:example "http://mywebsite.de/map.xodr" ;
+            sh:description "Link to traffic space"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of trafficSpace failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "trafficSpace"@en ;
+            sh:node general:LinkShape ;
+            sh:order 2 ;
+            sh:path scenario:trafficSpace ] ;
+    sh:targetClass scenario:Structural .
```

Comment:
Aren't those links covered by related or required data in general:General?

Reply:
Yeah this is definetly covered. It also becomes clearer with some improvements in the use of the required links. However, in a scenario there is a pretty tight coupling between the scenario and the map. I think with the general links you would have to really inspect each link to know which map this scenario refers to. Having a direct attribute would resemble the strong coupling and I think also reduce the lookup effort. 

So if it's not too bad, I'd like to have both.

Reply:
Okay, then let's have both